app/assets/javascripts/discourse/tests/integration/components/site-header-test.js

```diff
@@ -50,6 +50,33 @@ module("Integration | Component | site-header", function (hooks) {
     await click("header.d-header");
   });
 
+  test("rerenders when all_unread_notifications or unseen_reviewable_count change", async function (assert) {
+    this.currentUser.set("all_unread_notifications", 1);
+    this.currentUser.set("redesigned_user_menu_enabled", true);
```

Comment:
Instead of mixing the redesign menu test with the existing test, I think it'll make our job of cleaning up in the future easier if we move the tests into another file.

Reply:
Hmm so I create another file called something like `site-header-redesigned-menu-test.js`?

Reply:
Ah but I see we already have existing test for the redesigned menu. Maybe we can do the moving in another PR.